site/pt/blog/new-in-devtools-94/index.md

```diff
@@ -0,0 +1,177 @@
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/6dedde59f9d64290756a826f73dfe24cf382a470 #}
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/27aa364d1b194a7a778e7649e1f144abbed5957f #}
+
+Issue relacionada: [1179186](https://crbug.com/1179186)
+
+
+## Exibição consistente dos acessores nativos como propriedades no Console {: #native-accessors }
+
+O **Console** agora mostra acessores nativos como propriedades consistentemente.
+
+Por exemplo, ao avaliar a expressão `new Int8Array ([1, 2, 3])` no **Console**, acessadores nativos como `length`,` byteOffset` não eram exibidos na visualização. Com esta atualização, os acessores nativos aparecem na visualização e os valores são avaliados quando expandidos.
+
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/VcUiEcUXdWc00Q8595n6.png", alt="Exibição consistente dos acessores nativos como propriedades no Console", width="800", height="459" %}
+
+{# https://chromium.googlesource.com/v8/v8/+/ce8cef36aa7f386937a6b7bf1907e93b69cad1bd #}
+
+Issue relacionadas: [1076820](https://crbug.com/1076820), ​​[1199247](https://crbug.com/1199247)
+
+
+## Adequação dos stacks traces para scripts inline com #sourceURL {: #inline-script }
+
+O DevTools agora exibe scripts inline com `#sourceURL` corretamente e mostra os stack traces apropriados para depuração.
+
+Anteriormente, o DevTools exibia a localização incorreta para scripts inline com `#sourceURL`, em relação ao documento em vez de em relação à tag de abertura` <script>`.
+
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/XVUY8XxbGZW74kPsGkOZ.png", alt="Adequação dos stacks traces para scripts inline com #sourceURL", width="800", height="425" %}
+
+{# https://chromium.googlesource.com/v8/v8/+/c2f30c2b3f637c2339e8b9672c5c59a21b7d1095 #}
+
+Issue relacionadas: [1183990](https://crbug.com/1183990), ​​[578269](https://crbug.com/578269)
+
+## Alteração do formato da configuração da cor no painel Computed  {: #color-unit }
+
+Você agora pode alterar o formato da cor de qualquer elemento no painel Computed segurando a tecla <kbd>Shift</kbd> e clicando  na visualização da cor.
+
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/IhOkF5do9P8Ovlr7YsdX.png", alt="Shift+Clique na visualização da cor para alterar o formato da cor", width="800", height="474" %}
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/97143f7586d540e53a2e40ced7f106181e5c9ce3 #}
+
+Issue relacionada: [1226371](https://crbug.com/1226371)
+
+## Substituição dos tooltips customizados por tooltips nativos do HTML {: #tooltip }
+
+O DevTools agora adota tooltips nativos do HTML em todos os componentes. O DevTools teve uma implementação de tooltips customizados por muito tempo devido à falta de estilização de um tooltip nativo.
+
+Infelizmente, manter uma implementação de tooltips customizado é complicado e encontramos regularmente bugs complicados.
```

Comment:
- avoiding repetition of "complicado"
- "encontramos bugs complicados regularmente" seems to read a bit better.
```suggestion
Infelizmente, manter uma implementação de tooltips customizado é complexo e encontramos bugs complicados regularmente.
```